tables/enrol_request.rb

```diff
@@ -0,0 +1,53 @@
+class EnrolRequestTable < BaseTable
+  table_name 'enroll_requests'
+  scope { |ids| within_courses(ids) }
+
+  def migrate_batch(batch)
+    batch.each do |old|
+      new = ::Course::EnrolRequest.new
+
+      migrate(old, new) do
+        column :course_id do
+          store.get(V1::Course.table_name, old.course_id)
+        end
+        column :user_id do
+          result = store.get(V1::User.table_name, old.user_id)
```

Comment:
No need to two lines below, the result of the block will be assigned to `new.user_id`.

The syntax of the `column` helper` is explained below: 
```
column :name do
    foo()
end
# new.name = foo()

column :name
# new.name = old.name

column :old_name => :name
# new.name = old.old_name
```